Pear/Profile/ProfileSectionType.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ProfileSectionType.swift
+//  Pear
+//
+//  Created by Lucy Xu on 3/14/21.
+//  Copyright © 2021 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+enum ProfileSectionType {
+    case summary
+    case basics
+    case interests
+    case groups
+    case matches
+
+    func getReuseIdentifier() -> String {
```

Comment:
```suggestion
    var reuseIdentifier: String {
```

Make this a computed property. Ditto on line 29.